src/frontends/tensorflow_lite/src/op/densify.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "op_translation_utils.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "utils.hpp"
+
+using namespace std;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow_lite {
+namespace op {
+
+OutputVector densify(const ov::frontend::tensorflow_lite::NodeContext& node) {
+    auto input = node.get_input(0);
+    return {input};
```

Comment:
let us use existing translator `translate_identity_op` that performs identical action

Reply:
removed